Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if !WINDOWS_UWP
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    class PointerTests 
+    {
+        [SetUp]
+        public void Setup()
+        {
+            PlayModeTestUtilities.Setup();
+            TestUtilities.PlayspaceToOriginLookingForward();
+
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            PlayModeTestUtilities.TearDown();
+        }
+
+        #region Tests
+
+        /// <summary>
+        /// Tests that right after being instantiated, the pointer's direction 
+        /// is in the same general direction as the forward direction of the camera
+        /// </summary>
+        /// <returns></returns>
+        /// 
+        [UnityTest]
+        public IEnumerator TestPointerDirectionMatchesCameraDirectionFirstFrame()
+        {
+            var inputSystem = PlayModeTestUtilities.GetInputSystem();
+
+            // Raise the hand
+            var rightHand = new TestHand(Handedness.Right);
+
+            //Position 1 show hand
+            Vector3 initialPos = new Vector3(0.01f, 0.1f, 0.5f);
+            yield return rightHand.Show(initialPos);
+
+            //return first hand controller that is right and source type hand
```

Comment:
@julenka 
Added in:
```
            Vector3 linePointerOrigin = linePointer.Position;

            // Check that the line pointer origin is within half a centimeter of the initial position of the hand
            var distance = Vector3.Distance(initialPos, linePointerOrigin);
            Assert.LessOrEqual(distance, 0.005f);

```

The distance value comes out as 0.002237.  Also the linePointer.Position is the same as linePointer.Rays[0].Origin.  Which is better, linePointer.Position or linePointer.Rays[0].Origin?

Reply:
linePointer.Position is better because the rays might actually change some day (if we want to say make teh ray a little higher from the pointer's position)